### PR TITLE
Add convenience method to Pong for checking ping status.

### PR DESCRIPTION
**Pull request by egoebelbecker**
Added method. Fixed checkstyle checks.

### CODE REVIEW

**Comment by egoebelbecker** [comment]
Not sure if my changes caused these failures? It doesn't seem like they should have?

File: src/main/java/org/influxdb/dto/Pong.java [hunk 1]
```diff
@@ -25,6 +25,13 @@ public void setVersion(final String version) {
     this.version = version;
   }
 
+  /**
+   * Good or bad connection status.
+   */
+  public boolean isGood() {
+    return !"unknown".equalsIgnoreCase(version);
```
Comment (fmachado) [inline src/main/java/org/influxdb/dto/Pong.java]: @egoebelbecker would be possible to transform _"unknown"_ into a class scoped `static final` constant?

**Comment by fmachado** [comment]
CI failed for another reason that is not related with your changes.

File: src/main/java/org/influxdb/dto/Pong.java (hunk 1 above)
Comment (egoebelbecker) [reply src/main/java/org/influxdb/dto/Pong.java]: Good point. I have no idea why I didn't do that in the first place. Cancelling this and submitting a new one.

**Pull request merged**